superagi/resource_manager/file_manager.py

```diff
@@ -53,6 +54,14 @@ def write_to_s3(self, file_name, final_path):
                     s3_helper = S3Helper()
                     s3_helper.upload_file(img, path=resource.path)
 
+    def read_from_s3(self, file_path):
+        file_path = "resources" + file_path
+        logger.info(f"Reading file from s3: {file_path}")
+        response = S3Helper.get_s3_client().get_object(Bucket=get_config("BUCKET_NAME"), Key=file_path)
```

Comment:
use s3 helper as abstraction for all s3 methods. u can give these as object methods

Reply:
Done!